<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncounterLookup</class>
 <widget class="QWidget" name="EncounterLookup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encounter Lookup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelPokemon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Pok√©mon</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ComboBox" name="comboBoxPokemon"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelGame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Game</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ComboBox" name="comboBoxGame">
     <item>
      <property name="text">
       <string>Ruby</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sapphire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fire Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Leaf Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Emerald</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Diamond</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pearl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Platinum</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Heart Gold</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Soul Silver</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>White</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Black 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>White 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Brilliant Diamond</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shining Pearl</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonFind">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="TableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
